webservices/args.py

```diff
@@ -881,7 +881,9 @@ def make_seek_args(field=fields.Int, description=None):
     'min_dissemination_date': fields.Date(description=docs.DISSEMINATION_MIN_DATE),
     'max_dissemination_date': fields.Date(description=docs.DISSEMINATION_MAX_DATE),
     'min_filing_date': fields.Date(description=docs.FILED_DATE),
-    'max_filing_date': fields.Date(description=docs.FILED_DATE)
+    'max_filing_date': fields.Date(description=docs.FILED_DATE),
+    'most_recent': fields.Bool(description=docs.MOST_RECENT),
+
 }
 
```

Comment:
Could you create two separated doc variables like "MAX_FILE_DATE" and "MIN_FILE_DATE" for these two fields in docs.py? Thanks

Reply:
thanks @fec-jli. Yes, i have added a doc string to filing_date column